core-bundle/src/Resources/contao/models/OptInModel.php

```diff
@@ -166,6 +166,28 @@ public static function findByRelatedTableAndIds($strTable, array $arrIds, array
 		return static::createCollection($arrModels, $t);
 	}
 
+	/**
+	 * Count unconfirmed password reset opt-in tokens by their ID
+	 *
+	 * @param array $arrIds
+	 *
+	 * @return int
+	 *
+	 * @throws \Exception
+	 */
+	public static function countUnconfirmedPasswordResetTokensByIds(array $arrIds)
```

Comment:
```suggestion
	public static function countUnconfirmedPasswordResetTokensById($intId)
```

Why use an array if we ever only provide one ID?

Reply:
Adjusted in 6f61ac89cd.